hvplot/utilities.py

```diff
@@ -8,7 +10,7 @@
 
 save = _hv.save
 
-def show(obj, title=None, port=0, **kwargs):
+def show(obj, title=None, port=0, **kwargs) -> _pn.io.server.Server | _pn.io.server.StoppableThread:
```

Comment:
```suggestion
def show(obj, title=None, port=0, **kwargs):
```

Even though this is correct, hvplot is not yet ready to use type annotations.

Reply:
that's unfortunate, will remove those.